ROADMAP.md

```diff
@@ -0,0 +1,52 @@
+# wasmCloud Roadmap and Vision
+
+This document covers the current roadmap and vision for wasmCloud, as defined by the current project
+and org maintainers. Each category below has more information on what each thing means. As we start
+to define more of the items covered, we will link to the relevant issues. This may also continue to
+evolve as we use things like GitHub milestones or other tools for tracking our progres. 
+
+Overall, we want the community to have a good idea of where things are headed and what things they
+can contribute to. To be clear, this list is not meant as the end all, be all of what we are doing
+is wasmCloud. Not everything has to be on here to be considered "official work." It is meant more as
+a [guideline](https://tenor.com/bcCX3.gif) for what the maintainers are thinking about. If you don't
+see something on here that you think should be on here, feel free to [open a
+PR!](https://github.com/wasmCloud/wasmCloud/pulls)
+
+## Now
+
+This category contains things we are currently (or will very soon be) working on. By definition it
+is very narrow so you know exactly what is coming next 
+
+- SQL V2 contract
+- Finishing up [wadm](https://github.com/wasmCloud/wadm)
```

Comment:
If we can flesh these out a little bit, saying that we're doing V2 to further align with the Webassembly standard and protect against injection-type attacks that would be good

Re: wadm, could we add a few high level bullet points like improving the reconciliation algorithm, add a proper error state, and more that you've thought of?